src/main/resources/avro/referencemethods.avdl

```diff
@@ -261,17 +260,89 @@ ListReferenceBasesResponse getReferenceBases(
   /** Additional request parameters to restrict the query. */
   ListReferenceBasesRequest request) throws GAException;
 
-/****************  /sequence/{id}  *******************/
+/******************  /sequences/search  *********************/
+/**
+This request maps to the body of `POST /sequences/search` as JSON.
+*/
+record SearchSequencesRequest {
+  /**
+  If not null, return only `Sequence`s that appear in the indicated
+  `ReferenceSet`, or any included `ReferenceSet`s.
```

Comment:
See https://github.com/ga4gh/schemas/pull/250/files#diff-ffb7a8b69c62fd7062a6483e1e95d905R132 which @richarddurbin added.

Basically `ReferenceSet`s are also now supposed to be composeable: you can declare a new `ReferenceSet` as an old `ReferenceSet` plus some new `Sequence`s and `Join`s.